app/src/modules/settings/routes/appearance/item.vue

```diff
@@ -54,17 +53,12 @@ function discardAndLeave() {
 </script>
 
 <template>
-	<private-view :title="$t('settings_appearance')">
-		<template #headline><v-breadcrumb :items="[{ name: $t('settings'), to: '/settings' }]" /></template>
-		<template #title-outer:prepend>
-			<v-button class="header-icon" rounded icon exact disabled>
-				<v-icon name="palette" />
-			</v-button>
-		</template>
+	<private-view :title="$t('settings_appearance')" icon="palette">
+		<template #headline><v-breadcrumb :items="[{ name: t('settings'), to: '/settings' }]" /></template>
```

Comment:
This prevented the header from appearing as a side effect.

```suggestion
		<template #headline><v-breadcrumb :items="[{ name: $t('settings'), to: '/settings' }]" /></template>
```